sycl/test/sub_group/info.cpp

```diff
@@ -33,13 +33,31 @@ int main() {
         std::cout << "Expected exception has not been generated\n";
         return 1;
       } catch (runtime_error e) {
-        /* Expected exception - do nothing. */
+        try {
+          Device.get_info<info::device::sub_group_sizes>();
+          std::cout << "Expected exception has not been generated\n";
+          return 1;
+        } catch (runtime_error e) {
+          /* Expected exception - do nothing. */
+        }
       }
     }
 
   } else {
     Device.get_info<info::device::sub_group_independent_forward_progress>();
     Device.get_info<info::device::max_num_sub_groups>();
+    // There is no suport on CPU device for now
+    if (Device.get_info<info::device::device_type>() == info::device_type::cpu) {
```

Comment:
I believe this if should be true for device_type::accelerator too.

Reply:
done